tests/common/TestsSetup.hs

```diff
@@ -0,0 +1,42 @@
+-----------------------------------------------------------------------------
+--
+-- Module      :  Main
+-- Copyright   :  (c) Oleg Grenrus 2015
```

Comment:
I've written up the new approach, which is hopefully as simple as possible, here:
https://github.com/purescript/purescript/issues/1394